README.md

```diff
@@ -283,6 +283,8 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+Joan Lloret(JDLloret):| I tried so hard and got it wrong
+
```

Comment:
Then in order to finish you must place the link in there, by placing 
`[nickname] : <link_to_github_profile>` and important in alphabeticall order.